### PR TITLE
Add ARIA attributes for components

### PR DESCRIPTION
**Pull request by harishkumar-srijan**

### CODE REVIEW

File: stories/Atom/Typography/Links/link.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import './link.scss';
 
 export const Link = ({ label }) => (
-  <a href="#">
+  <a href="#" role="link" alt={label} title={label}>
```
Comment (surbhi90) [inline stories/Atom/Typography/Links/link.jsx:5]: @harishkumar-srijan i don't think alt is the attribute of anchor tag and do we need the role='link' attribute on anchor

File: stories/Components/Navigationcomponents/Menu/Menu.jsx [hunk 2]
```diff
@@ -3,9 +3,11 @@ import './menu.scss';
 import { MenuItems } from '../../../Atom/Navigation/Menu-items/Menu-Items';
 
 export const Menu = ({ data }) => (
-  <ul className="menu">
-    {data.map((item, index) => (
-      <li key={index} id={item.id}><MenuItems text={item.label} /></li>
-    ))}
-  </ul>
+  <div role="navigation" aria-label="menu">
```
Comment (surbhi90) [inline stories/Components/Navigationcomponents/Menu/Menu.jsx:6]: @harishkumar-srijan can't we use nav tag here

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Atom/Typography/Links/link.jsx (hunk 1 above)
Comment (harishkumar-srijan) [reply stories/Atom/Typography/Links/link.jsx]: done

File: stories/Components/Navigationcomponents/Menu/Menu.jsx (hunk 2 above)
Comment (harishkumar-srijan) [reply stories/Components/Navigationcomponents/Menu/Menu.jsx]: done

**Review by surbhi90 — APPROVED** [review]

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**